test/test-tools.go

```diff
@@ -174,7 +174,12 @@ func CountCounter(counter prometheus.Counter) int {
 	return int(iom.Counter.GetValue())
 }
 
-func CountHistogramSamples(hist prometheus.Histogram) int {
+func CountHistogramSamples(obs prometheus.Observer) int {
+	hist, ok := obs.(prometheus.Histogram)
+	if !ok {
```

Comment:
If we want to panic, there's no need to use the `, ok` formulation. A type assertion without `, ok` will just panic if it fails.

Reply:
Good point. It won't have as descriptive an error message but I think that's fine. Will change.

Reply:
Changed in 79ceaa6